server/src/routes/api/v1/moviesRouter.js

```diff
@@ -17,7 +18,15 @@ moviesRouter.get("/:id", async (req, res) => {
   const id = req.params.id;
   try {
     const movie = await Movie.query().findById(id);
-    const serializedMovie = await MovieSerializer.getDetails(movie);
+    const serializedMovie = MovieSerializer.getDetails(movie);
+
+    const reviews = await movie.$relatedQuery("reviews");
+    const serializedReviews = reviews.map((review) => {
```

Comment:
consider moving this related query and review serialization into the MovieSerializer when you getDetails